angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html

```diff
@@ -3,67 +3,74 @@
   <app-header app-title="Browse {{NAMES.title}}"
               app-icon="fa-home">
   </app-header>
-  <div class="mp-header">
-    <input type="text" class="marketplace-search" placeholder="What are you looking for?" ng-model="searchText" select-on-page-load aria-label="Search bar enter the app you are looking for">
-  </div>
-  <div class="mp-opaque">
-    <ul>
-      <li ng-click="selectFilter('popular','')" ng-class="{true:'selected-filter'}[selectedFilter === 'popular']"><a href="#" aria-label="search from the most popular apps">Most Popular</a></li>
-      <li ng-click="selectFilter('az','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'az']"><a href="#" aria-label="search by alphabetical order">A-Z</a></li>
-      <li ng-click="selectFilter('category','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'category']"><a href="#" aria-label="search by categories">Categories</a></li>
-    </ul>
-  </div>
-  <div class="show-all-div">
-    <p><span ng-if="portlets.length > 0">Showing {{ (portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length }} of {{ portlets.length }} results in {{NAMES.title}}.</span>
-    <span ng-if="portlets.length == 0">Loading...</span></p>
-    <div ng-hide="GuestMode"><p><input type="checkbox" ng-click="toggleShowAll()" ng-checked="showAll">Show all<span popover="Show all apps, even if I don't have access" popover-append-to-body="true" popover-placement="left" popover-trigger="mouseenter"><i class="fa fa-question-circle" ></i></span></p></input></div>
-  </div>
-  <div ng-show="showCategories" class="show-categories">
-    <p>Categories:</p><a ng-repeat="category in categories" href="" class="category-links" ng-click="selectFilter('category',category)" ng-class="{true: 'selected-category'}[categoryToShow === category]">{{category}}</a>
-  </div>
+  <div class="marketplace-body">
+    <div class="mp-header">
+      <div class="browse-help">
+        <p class="hidden-xs">What are you looking for?</p>
+        <p class="visible-xs"><i class="fa fa-search"></i></p>
+      </div>
+      <div class="browse-input">
+        <input type="text" class="hidden-xs marketplace-search" placeholder="e.g. Benefits Information" ng-model="searchText" select-on-page-load aria-label="Search bar enter the app you are looking for">
```

Comment:
Note that this is a Madison-ism coded into the `marketplace.html` template. Not a deal-breaker, but as we continue to encourage adoption of `angularjs-portal` beyond MyUW, need a strategy for localization more compelling than forking the HTML templates. Viable text externalization strategies?

This specific instance is interesting in that it might be worthwhile to externalize this into a set of example, so that sometimes it would say "e.g. Benefits Information" but sometimes it would say "e.g. Courses".

Which leads to the idea that maybe this should be "e.g. {randomly selected name of app from set of apps the viewing user can both browse and subscribe}".  

Which is scope creep in a lot of ways, but also solves the problem that the "Benefits Information" example is irrelevant to the non-employees in our user base.